Assets/VRTK/Scripts/VRTK_Hip_Tracking.cs

```diff
@@ -0,0 +1,73 @@
+﻿namespace VRTK
+{
+	using UnityEngine;
+	using System.Collections;
+
+	public class VRTK_Hip_Tracking : MonoBehaviour {
+
+		/// <summary>
+    	/// Hip Tracking attempts to reasonably track hip position in the absence of a hip position sensor
+    	/// </summary>
+   	 	/// <remarks>
+		///
+		/// The Hip Tracking script is placed on an empty GameObject which will be positioned at the estimated hip position
+		/// <para>&#160;</para>
+		/// Variables
```

Comment:
You don't need to do the variables here, you can just add tooltips, the inspector params gets built off of those.